tmt/steps/execute/__init__.py

```diff
@@ -597,16 +680,18 @@ def prepare_tests(self, guest: Guest, logger: tmt.log.Logger) -> list[TestInvoca
 
     def prepare_scripts(self, guest: "tmt.steps.provision.Guest") -> None:
         """ Prepare additional scripts for testing """
+        # Create scripts directory
+        guest.execute(Command("mkdir", "-p", str(effective_scripts_dest_dir())))
```

Comment:
How about using rsync with `--mkpath` ?  
Actually, there is also `--executability` or `--perms`/`--chmod` to get rid of the other ssh command.  
(again, can be a separate PR)

Reply:
@martinhoyer rsync --mkpath was added in rsync 3.2.3 (6 Aug 2020). I wonder if it will work on all supported OSes

Reply:
I think I did check that before writing the comment.

Reply:
That option has to exist for rsync on the guest as well, right? If so we definitely _cannot_ use that as rhel-7 has just rsync-3.1.2

Reply:
hmm, yeah I also see el8 is too old as well. Alas.